 * Serving Flask app 'app'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8083
 * Running on http://172.19.78.144:8083
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
127.0.0.1 - - [22/May/2025 15:34:34] "GET /tablas/ HTTP/1.1" 200 -
127.0.0.1 - - [22/May/2025 15:34:35] "GET /tablas/generar_tabla HTTP/1.1" 200 -
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\tablas_busqueda.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\tablas_busqueda.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\tablas_busqueda.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
127.0.0.1 - - [22/May/2025 15:44:33] "GET /tablas/ HTTP/1.1" 200 -
127.0.0.1 - - [22/May/2025 15:44:34] "GET /tablas/generar_tabla HTTP/1.1" 500 -
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\tablas_busqueda.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
127.0.0.1 - - [22/May/2025 16:11:24] "GET /tablas/generar_tabla HTTP/1.1" 500 -
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\.ipynb_checkpoints\\tablas_busqueda-checkpoint.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
127.0.0.1 - - [22/May/2025 16:11:37] "GET /tablas/generar_tabla HTTP/1.1" 200 -
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\tablas_busqueda.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 303-598-026
127.0.0.1 - - [22/May/2025 16:13:12] "GET /tablas/ HTTP/1.1" 200 -
127.0.0.1 - - [22/May/2025 16:13:13] "GET /tablas/generar_tabla HTTP/1.1" 200 -
127.0.0.1 - - [22/May/2025 16:14:12] "GET /tablas/ HTTP/1.1" 200 -
127.0.0.1 - - [22/May/2025 16:14:13] "GET /tablas/generar_tabla HTTP/1.1" 200 -
 * Detected change in 'C:\\Users\\francisco.contreras\\Desktop\\jupyter-projects\\notebooks\\MAYA-Modular\\app\\main.py', reloading
 * Restarting with stat
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\francisco.contreras\Desktop\jupyter-projects\notebooks\MAYA-Modular\app\main.py", line 4, in <module>
    from app.functions import extract_and_store_documents
ModuleNotFoundError: No module named 'app.functions'
